File: Day 2 : Operators/Solution.py
def solve(meal_cost, tip_percent, tax_percent):
    tip = (tip_percent / 100) * meal_cost
    tax = (tax_percent / 100) * meal_cost
    total = meal_cost + tip + tax
    print(round(total))

if __name__ == '__main__':
    #input all the variables
    meal_cost = float(input().strip())

    tip_percent = int(input().strip())

    tax_percent = int(input().strip())

    solve(meal_cost, tip_percent, tax_percent)

File: Day 2 : Operators/Solution.c
#include <stdio.h>

void solve(double meal_cost, int tip_percent, int tax_percent) {
    double tip, tax, total;
    tip = ((double)tip_percent / 100) * meal_cost;
    tax = ((double)tax_percent / 100) * meal_cost;
    total = meal_cost + tip + tax;
    printf("%.0lf", total);
}

int main()
{
    double meal_cost;
    int tip_percent;
    int tax_percent;

    scanf("%lf", &meal_cost);
    scanf("%d", &tip_percent);
    scanf("%d", &tax_percent);
    solve(meal_cost, tip_percent, tax_percent);

    return 0;
}
